src/test/versionbits_tests.cpp

```diff
@@ -32,6 +32,12 @@ class TestConditionChecker : public AbstractThresholdConditionChecker
     int GetStateSinceHeightFor(const CBlockIndex* pindexPrev) const { return AbstractThresholdConditionChecker::GetStateSinceHeightFor(pindexPrev, paramsDummy, cache); }
 };
 
+class TestFixedHeightConditionChecker : public TestConditionChecker
```

Comment:
nit: "FixedHeight" should be "AlwaysActive" or similar, everywhere in this file.

Reply:
Done, renamed in many places.